### PR TITLE
[Vagrantfile] Update used box to Ubuntu 14.04

### PR DESCRIPTION
**Pull request by berendt**

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -41,7 +41,7 @@ source /etc/profile.d/gopath.sh
 SCRIPT
 
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
-  config.vm.box = "chef/ubuntu-12.04"
+  config.vm.box = "ubuntu/trusty64"
```
Comment (catsby) [inline Vagrantfile]: The [Trusty64 box from Ubuntu][1] hosted on Atlas does not support the configured provider below (vmware). Is there a particular advantage to using Trusty here? 

[1]: https://atlas.hashicorp.com/ubuntu/boxes/trusty64

Comment (berendt) [inline Vagrantfile]: After starting Ubuntu 12.04 there is a note that I should uprade to 14.04. This is annoying. Also I thought it makes sense to use the official Vagrant boxes provided by the Ubuntu project. I was not aware that they do not provide VMware boxes. What about using [chef/ubuntu-14.04](https://atlas.hashicorp.com/chef/boxes/ubuntu-14.04)?

**Comment by berendt** [comment]
Rebased and changed Vagrantbox to `chef/ubuntu-14.04`.

**Comment by berendt** [comment]
Anything else to do here at the moment?

**Comment by jen20** [comment]
These have moved to the `bento` organization, but are still on 12.04 - I'm not sure there is any harm in moving to 14.04 at some point though.

**Pull request closed without merging**